cmd/boulder-wfe2/main.go

```diff
@@ -165,6 +165,12 @@ type Config struct {
 		// CP/CPS, under "DV-SSL Subscriber Certificate". The value must match
 		// the CA and RA configurations.
 		MaxNames int `validate:"min=0,max=100"`
+
+		// CertificateProfileNames is the list of acceptable certificate profile
+		// names for newOrder requests. Requests with a profile name not in this
+		// list will be rejected. This field is optional; if unset, all profile
+		// names are accepted.
```

Comment:
Would we consider saying "if unset, no profile names are propagated forward"? I think that failing closed -- not storing arbitrary query parameters in the database just because a config field is missing -- seems safer to me.

Reply:
Seems safer to me as well. I'll go ahead and make the change.